BuildingSync.xsd

```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="DemisingAboveGradeWallArea" minOccurs="0">
```

Comment:
I propose changing the element name to: `TotalCommonConditionedAboveGradeWallArea` and the documentation to: `The total area of the above grade wall common with other conditioned buildings`